go.mod

```diff
@@ -5,6 +5,7 @@ go 1.18
 require (
 	github.com/AdaLogics/go-fuzz-headers v0.0.0-20220706123152-fef3fe1bab07
 	github.com/Microsoft/go-winio v0.5.2
+	github.com/agiledragon/gomonkey/v2 v2.8.0
```

Comment:
I’m not sure we want to introduce this dependency.
Probably hard to debug potential issues

Reply:
This is just for auxiliary unit testing, without this, I can't think of any other way to mock some functions inside.